src/pages/components/leadspace.mdx

```diff
@@ -41,26 +55,27 @@ This option has 4 empty columns on the left and right to accommodate pages that
+| Headline       | Short copy      | Yes                                   | XS (65 / 85)                                   | –               |
+| Description    | Short copy      | No                                    | M (120 / 150)                                  | –               |
+| [Button group](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/button-group) | Component | No | XXS (40 / 55) | Lead space Short (320px) does not allow a Button group or a short copy. |
+| Gradient       | True/False      | No                                    | –         | True – Show gradient (default) <br/>False – No gradient |
+| Background [image](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/image) | Component | No | –  | Breakpoints require different images. |
+
 
 ### Lead space centered
 
-| Field | Field type | Required   | Cardinality | Max character size <br/>(English / translated) | Notes |
-| -------- | --------- | ---------- | ---------- | ----------| ----------|
-| Headline | Short copy | Yes | 1 | XS (65 / 85)| - |
-| Description | Short copy | No | 1 | M (120 / 150)| - |
-| [Button group](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/button-group) | Component | No | Max of two buttons | - | - |
-| [Image](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/image) | Component | No | 1 | - | - |
-| Gradient | True/False | Yes | - | - | True - Show gradient (default)<br/>False - No gradient |
-| Type | Option list | Yes | - | - | Center, Left, or Small |
+| Field              | Field type      | Required     | Cardinality       | Max character size <br/>(English / translated)        | Notes          |
+| ------------------ | --------------- | ------------ | ----------------- | ------------------------------------------------------| ---------------|
+| Headline           | Short copy      | Yes          | 1                 | XS (65 / 85)                                          | –              |
+| Description        | Short copy      | No           | 1                 | M (120 / 150)                                         | –              |
+| [Button group](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/button-group) | Component | No | Max of two buttons | XXS (40 / 55) | –    |
+| Gradient           | True/False      | No           | –                 | –                                                     | True - Show gradient (default)<br/>False - No gradient |
+| Background [image](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/image)               | Component | No | 1                  | – | Breakpoints require different images.    |
+| Type               | Option list     | Yes          | –                 | –                                                     | Center, Left, or Small |
```

Comment:
Are we overcomplicating things by providing two separate content models for a single component? I feel look at consolidating.

Reply:
We can definitely look at this. I already started it, but the stringent need for a shorter height and previous research already in place topped it.